arch/x86/soc/apollo_lake/soc.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2010-2015, Wind River Systems, Inc.
```

Comment:
why this line is needed?

Reply:
The file was copied from the arch/x86/soc/atom. So need to preserve copyright.

Reply:
You should put your copyright on top if you have made any changes